marc.py

```diff
@@ -560,11 +560,13 @@ def get_storage_settings(cls, _db):
         for integration in integrations:
             # Only add an integration to choose from if it has a 
             # MARC File Bucket field in its settings.
-            [configuration_setting] = [s for s in integration.settings if s.key=="marc_bucket"]
-            if configuration_setting.value:
-                cls.SETTING['options'].append(
-                    dict(key=str(integration.id), label=integration.name)
-                )
+            configuration_settings = [s for s in integration.settings if s.key=="marc_bucket"]
+
+            if configuration_settings:
+                if configuration_settings[0].value:
+                    cls.SETTING['options'].append(
+                        dict(key=str(integration.id), label=integration.name)
+                    )
```

Comment:
How can it happen that a single integration has multiple values for `marc_bucket`?

Reply:
It can't. This piece of code checks whether this configuration setting exists.
It's required because I changed the number of settings in `LCPMirror` defined in `circulation`. It doesn't have `marc_bucket` setting

Reply:
I see, so a single integration might have _less_ than one value for `marc_bucket`.